### ISSUE TITLE
Why loss increase with the increase of training times when training realnoisy

### ISSUE THREAD

**Issue by Suntory-Rakuzen** [scut-mingqinchen/self2self#9]
When I training the realnoisy , the picture will be blacker , and the loss will be bigger.And the 1000 training steps result seems good ,but when training times get bigger, the result becomes worser and the noise appears
